pkg/reconciler/reconciler.go

```diff
@@ -257,6 +260,17 @@ func SkipDependentWatches(skip bool) Option {
 	}
 }
 
+// SkipSchemeSetup is an Option that configures whether the the Manager's
+// Scheme is to be set up in a generic way as part of the Reconciler setup.
+//
+// By default, skipping of the scheme setup is disabled.
```

Comment:
Nit: maybe slightly easier to grok wording.
```suggestion
// By default, scheme setup will not be skipped.
```